CONTRIBUTING.md

```diff
@@ -1128,7 +1128,7 @@ have more checks than older versions. In our CI, we run clang-tidy-6.0.
    uncommitted changes). Changes are picked up based on a `git diff` with the
    given revision:
   ```bash
-  python tools/linter/clang_tidy.py -d build -p torch/csrc --diff 'HEAD~1'
+  python tools/linter/clang_tidy -d build -p torch/csrc --diff 'HEAD~1'
```

Comment:
when I run this, I get the following error:
```
clang_tidy: error: unrecognized arguments: -d
```

Reply:
Good catch, the docs need to be updated. The flags have changed.

Reply:
Ah I forgot to update this in #60048, thanks for fixing it!

Reply:
You probably already know this, but when I removed the -d flag, the next error was

`FileNotFoundError: [Errno 2] No such file or directory: 'HEAD~1'`